doc/en/user/source/community/wps-download/rawDownload.rst

```diff
@@ -378,3 +379,42 @@ This time, when issued (and process has finished on the server), the GET request
       </wps:Output>
     </wps:ProcessOutputs>
   </wps:ExecuteResponse>
+
+
+Writing parameters customization
+++++++++++++++++++++++++++++++++
+
+The ``writeParameters`` input element of a process execution allows to specify parameters to be applied by the ``outputFormat`` encoder when producing the output file.
+Writing parameters are listed as multiple ``<dwn:Parameter key="writingParameterName">value</dwn:Parameter>`` within a ``<dwn:Parameters>`` parent element.
+See the below xml containing full syntax of a valid example for TIFF output format:
+
+.. code-block:: xml
+
+    <wps:Input>
+      <ows:Identifier>writeParameters</ows:Identifier>
+        <wps:Data>
+           <wps:ComplexData xmlns:dwn="http://geoserver.org/wps/download">
+             <dwn:Parameters>
+                <dwn:Parameter key="tilewidth">128</dwn:Parameter>
+                <dwn:Parameter key="tileheight">128</dwn:Parameter>
+                <dwn:Parameter key="compression">JPEG</dwn:Parameter>
+                <dwn:Parameter key="quality">0.75</dwn:Parameter>
+             </dwn:Parameters>
+           </wps:ComplexData>
+        </wps:Data>
+    </wps:Input>
+
+GeoTIFF/TIFF supported writing parameters
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The supported writing parameters are:
+
+ * ``tilewidth`` : Width of internal tiles, in pixels
+ * ``tileheight`` : Height of internal tiles, in pixels
+ * ``compression`` : Compression type used to store internal tiles. Most commonly used values are:
```

Comment:
I'd write down the actual supported list (you can check the GeoTiff WCS 2.0 support for a more complete one)